src/Avalonia.Themes.Fluent/ColorPaletteResources.Properties.cs

```diff
@@ -0,0 +1,153 @@
+﻿using Avalonia.Media;
+
+namespace Avalonia.Themes.Fluent;
+
+public partial class ColorPaletteResources
+{
+    private bool _hasAccentColor;
+    private Color _accentColor;
+    private Color _accentColorDark1, _accentColorDark2, _accentColorDark3;
+    private Color _accentColorLight1, _accentColorLight2, _accentColorLight3;
+
+    public static readonly DirectProperty<ColorPaletteResources, Color> AccentProperty
+        = AvaloniaProperty.RegisterDirect<ColorPaletteResources, Color>(nameof(Accent), r => r.Accent, (r, v) => r.Accent = v);
+
+    /// <summary>
+    /// Gets or sets the Accent color value.
+    /// </summary>
+    public Color Accent
+    {
+        get => _accentColor;
+        set => SetAndRaise(AccentProperty, ref _accentColor, value);
+    }
+
+    /// <summary>
+    /// Gets or sets the AltHigh color value.
+    /// </summary>
+    public Color AltHigh { get => GetColor("SystemAltHighColor"); set => SetColor("SystemAltHighColor", value); }
```

Comment:
Only accent color is bindable, as it is used with DynamicResource everywhere.
Other colors are not bindable, and I consider adding some kind of setter validation to throw an exception, if it was setter after resources were read once.